src/connectors/powered-by/connectPoweredBy.js

```diff
@@ -30,20 +40,29 @@ export default function connectPoweredBy(renderFn, unmountFn) {
   checkRendering(renderFn, usage);
 
   return (widgetParams = {}) => {
+    const defaultUrl =
+      'https://www.algolia.com/?' +
+      'utm_source=instantsearch.js&' +
+      'utm_medium=website&' +
+      `utm_content=${location.hostname}&` +
+      'utm_campaign=poweredby';
+
     return {
-      init() {
+      init({ url = defaultUrl }) {
```

Comment:
The URL should come from the widgetParams not the InstantSearch instance to allow other users to override it. Right now we don't have a way to override it from user code. For example in Vue we should be able to do something like:

```js
export default {
  mixins: [createWidget({ connector: connectPoweredBy })],
  computed: {
    widgetParams() {
      return {
        // it would be nice to also support this:
        source: "vue-instantsearch",
        // but also
        URL:
          "https://www.algolia.com/?" +
          "utm_source=vue-instantsearch&" +
          "utm_medium=website&" +
          `utm_content=${location ? location.hostname : ""}&` +
          "utm_campaign=poweredby"
      };
    }
  }
};
```